app/controllers/concerns/idv/ab_test_analytics_concern.rb

```diff
@@ -0,0 +1,11 @@
+module Idv
+  module AbTestAnalyticsConcern
+    include AcuantConcern
+    include Idv::GettingStartedAbTestConcern
```

Comment:
I start getting a little nervous when I see a concern the includes a concern that includes a concern.

I'm thinking through whether there is some way for us to flatten this out into an A/B test concern that includes methods for each of the A/B tests we are performing and the tooling for logging them 🤔. That helps to make the rabbit a little more shallow.

Reply:
We could put just the analytics methods in here and not include the rest of the concern. I started that way and then thought this might be neater.

Reply:
That didn't work out at all! I think we'd have to combine all the code for the A/B tests in one Concern, like you said, and that doesn't seem great either.